test/test_pipeline/test_classification.py

```diff
@@ -206,7 +206,7 @@ def test_configurations_categorical_data(self):
             dataset_properties={'sparse': False},
             include={
                 'preprocessor': ['no_preprocessing'],
-                'classifier': ['sgd', 'gradient_boosting']
+                'classifier': ['sgd', 'adaboost']
```

Comment:
@mfeurer : this test was failing because the new gradient_boosting currently doesn't support sample_weight. Since this is a test of the classification pipeline (and not of a specific classifier), I thought that the quickest fix would be to just pick another classifier with support for sample_weight. Hope that's alright.

Reply:
Yes, that makes sense.